docs/cli_installation.md

```diff
@@ -81,7 +81,7 @@ $version = (Invoke-RestMethod https://api.github.com/repos/argoproj/argo-cd/rele
 Replace `$version` in the command below with the version of Argo CD you would like to download:
 
 ```powershell
-$url = "https://github.com/argoproj/argo-cd/releases/download/" + $version + "/argocd-windows-amd64.exe"
+$url = "https://github.com/argoproj/argo-cd/releases/download/" + '$version' + "/argocd-windows-amd64.exe"
```

Comment:
@adefemi171 thanks for the contribution! I'm unfamiliar with powershell. What's the impact of this change?

Reply:
This would stop the version from being concatenated into the URL. I don't see a reason to do this. Closing